tests/sim/test_sim_pim.py

```diff
@@ -174,3 +174,12 @@ def test_PIM_noise():
     status = pim.move("IN")
     assert(status.success)
     assert(pim.sim_y.value != 5 and np.isclose(pim.sim_y.value, 5, atol=0.1))
+
+def test_PIMMotor_stage():
+    pim = PIMMotor("TEST", pos_in=5)
+    pim.move_out(wait=True)
+    pim.stage()
+    assert(pim.position == "IN")
+    pim.unstage()
+    time.sleep(0.2)
```

Comment:
.unstage() does not wait, I have no callback for completion, and I'm not guaranteed that the simulated api won't change to do some sort of asynchronous movement later. The sleep in the sim move was 0.1s, so I picked 0.2s. If there's something clever I can do with subscribe I can switch to that instead.